tests/units/classes/test.php

```diff
@@ -466,6 +479,22 @@ public function testGetTestedClassPath()
 			;
 		}
 
+		public function testCallStaticOnTestedClass()
+		{
+			$this
+				->if($this->setTestedClassName('\mageekguy\atoum\withStatic'))
+				->then
+					->string($this->callStaticOnTestedClass('staticMethod', $return = uniqid()))->isEqualTo($return)
```

Comment:
Thanks for the answer @jubianchi 
It seems strange to me to do a fake test to test the test tool itself ôO

But i do it.

Thanks ;)